### ISSUE TITLE
Common.js release on Bower

### ISSUE THREAD

**Issue by beckyconning** [kjur/jsrsasign#155]
It would be nice to have a CommonJS module release on Bower instead of just on NPM for those of us using Purescript, Webpack or Browserify : ).

Thanks!

**Comment by GuillaumeUnice** [comment]
+1

**Comment by kjur** [comment]
I don't know how to support it. I wish I would have pull request for it :(

**Comment by maxmantz** [comment]
+1 for this. 

I would prefer a UMD release though for it to be universally usable. 

Sadly the npm version is 270kb in size and you will have to load everything even if you just need a small portion of the library.

I would love to help. I've already tried modifying the parts I need by adding a `module.exports` statement at the bottom of each file, exporting the global variable created by the file. However all files and all of their dependencies also heavily rely on global variables. Alone I will have no chance figuring out which exports which.

Basically the steps required for each library file are:
- export the global variable created by the file using `module.exports = VARNAME;`
- import every required global variable using `var dependeny = require('path/to/file');`

Once this is in place we can think about a webpack configuration pulling it all together.

**Comment by kjur** [comment]
I've released 6.0.0 today which is now Node.js and "fs" safe. Please check it.
